dockerfiles/Dockerfile.test-gunicorn

```diff
@@ -0,0 +1,34 @@
+FROM --platform=linux/amd64 quay.io/sustainable_computing_io/kepler_model_server_base:v0.7
+
+WORKDIR /usr/local
+
+RUN mkdir -p /usr/local/src
+RUN mkdir -p /usr/local/resource
+
+COPY dockerfiles/requirements.txt dockerfiles/requirements.txt
+RUN pip install --no-cache-dir -r dockerfiles/requirements.txt
+
+COPY src/estimate src/estimate
+COPY src/server src/server
+COPY src/train src/train
+COPY src/util src/util
+COPY cmd cmd
```

Comment:
why not copy the entire src and cmd ?

Reply:
I think it is my bad that when we run the test the built ./model will be inside the src folder. We need to fix that first then the src will be cleaned enough for entire copy.

Reply:
I can check it out.